ttnn/cpp/ttnn/operations/conv/conv2d/conv2d_utils.cpp

```diff
@@ -5,19 +5,24 @@
 #include <sys/types.h>
 #include <cstdint>
 #include <optional>
+#include <tuple>
 
 #include "conv2d_utils.hpp"
 #include "common/constants.hpp"
+#include "common/tt_backend_api_types.hpp"
 #include "impl/buffers/buffer_constants.hpp"
+#include "ttnn/operations/conv/conv2d/device/conv2d_op.hpp"
```

Comment:
this shoudn't be included here?

Reply:
`estimate_L1_usage` and `Conv2dConfig` are defined in `device/conv2d_op.hpp`